cli/lsp/language_server.rs

```diff
@@ -137,6 +138,7 @@ pub struct LanguageServer {
   init_flag: AsyncFlag,
   performance: Arc<Performance>,
   shutdown_flag: AsyncFlag,
+  worker_runtime: Arc<tokio::runtime::Runtime>,
```

Comment:
Is this purely for running cancellation tokens? This could use some comments explaining the reason for it